Traceback (most recent call last):
  File "/home/sergio/ros2_ws/build/navigation_tb3/ament_cmake_python/navigation_tb3/setup.py", line 1, in <module>
    from setuptools import find_packages
  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 16, in <module>
    import setuptools.version
  File "/usr/lib/python3/dist-packages/setuptools/version.py", line 1, in <module>
    import pkg_resources
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3267, in <module>
    def _initialize_master_working_set():
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3241, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3302, in _initialize_master_working_set
    list(map(working_set.add_entry, sys.path))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 613, in add_entry
    for dist in find_distributions(entry, True):
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2077, in find_on_path
    for dist in factory(fullpath):
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2142, in distributions_from_metadata
    yield Distribution.from_location(
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2606, in from_location
    )._reload_version()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3004, in _reload_version
    md_version = self._get_version()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2785, in _get_version
    version = _version_from_file(lines)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2569, in _version_from_file
    line = next(iter(version_lines), '')
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2780, in _get_metadata
    for line in self.get_metadata_lines(name):
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1422, in get_metadata_lines
    return yield_lines(self.get_metadata(name))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1412, in get_metadata
    value = self._get(path)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1616, in _get
    with open(path, 'rb') as stream:
FileNotFoundError: [Errno 2] No such file or directory: '/home/sergio/ros2_ws/install/mma_listener/lib/python3.10/site-packages/mma_listener-0.0.0-py3.10.egg-info/PKG-INFO'
gmake[2]: *** [CMakeFiles/ament_cmake_python_build_navigation_tb3_egg.dir/build.make:70: CMakeFiles/ament_cmake_python_build_navigation_tb3_egg] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:192: CMakeFiles/ament_cmake_python_build_navigation_tb3_egg.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
